alternator/executor.cc

```diff
@@ -2300,22 +2300,22 @@ static std::optional<attrs_to_get> calculate_attrs_to_get(const rjson::value& re
  * as before.
  */ 
 void executor::describe_single_item(const cql3::selection::selection& selection,
-    const std::vector<bytes_opt>& result_row,
+    const std::vector<managed_bytes_opt>& result_row,
     const std::optional<attrs_to_get>& attrs_to_get,
     rjson::value& item,
     bool include_all_embedded_attributes) 
 {
     const auto& columns = selection.get_columns();
     auto column_it = columns.begin();
-    for (const bytes_opt& cell : result_row) {
+    for (const managed_bytes_opt& cell : result_row) {
         std::string column_name = (*column_it)->name_as_text();
         if (cell && column_name != executor::ATTRS_COLUMN_NAME) {
             if (!attrs_to_get || attrs_to_get->contains(column_name)) {
                 // item is expected to start empty, and column_name are unique
                 // so add() makes sense
                 rjson::add_with_string_name(item, column_name, rjson::empty_object());
                 rjson::value& field = item[column_name.c_str()];
-                rjson::add_with_string_name(field, type_to_string((*column_it)->type), json_key_column_value(*cell, **column_it));
+                rjson::add_with_string_name(field, type_to_string((*column_it)->type), json_key_column_value(to_bytes(*cell), **column_it));
```

Comment:
Can you use here to_bytes_view? to_bytes will needlessly make a copy :-(

Reply:
Right.... I mis-remembered that you had a to_bytes_view() function that does this opportunistic avoiding copying, but now I realize that it's impossible to implement because it can't return a pointer to a temporary copy without storing the copy separately...